### DISCUSSION TITLE
Welcome to pygeometry Discussions!

### DISCUSSION THREAD

**Discussion by akaszynski** [ansys/pyansys-geometry#86] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TakaYM** [comment]
There is .run_discovery_script_file for Discovery. But I can not see for SpaceClaim API. Can we run SpaceClaim with  .run_discovery_script_file ?

**Reply by RobPasMue** [reply]
Hi @TakaYM! Yes, you can run the scripting API for SpaceClaim through that same endpoint.

**Reply by TakaYM** [reply]
Thanks @RobPasMue for your quick advice!
